typescript/lib/parsers/openai.ts

```diff
@@ -274,7 +276,7 @@ export class OpenAIChatModelParser extends ParameterizedModelParser<Chat.ChatCom
     } else if (typeof prompt.input?.data === "string") {
       return prompt.input?.data;
     } else {
-      const message = prompt.input as Chat.ChatCompletionMessageParam;
+      const message = prompt.input?.data as Chat.ChatCompletionMessageParam;
```

Comment:
Oh, nice fix

Reply:
Oh wait, not a fix. The intention is to support raw dump of completion params in the input instead of under .data